x-pack/plugins/reporting/public/lib/reporting_api_client/reporting_api_client.ts

```diff
@@ -175,7 +176,7 @@ export class ReportingAPIClient implements IReportingAPI {
   public async createReportingJob(exportType: string, jobParams: BaseParams) {
     const jobParamsRison = rison.encode(jobParams);
     const resp: { job: ReportApiJSON } = await this.http.post(
-      `${INTERNAL_ROUTES.GENERATE_PREFIX}/${exportType}`,
+      `${INTERNAL_ROUTES.GENERATE_PREFIX}/${exportType}?${ELASTIC_INTERNAL_ORIGIN_QUERY_PARAM}=true`,
```

Comment:
Was this just something missed from https://github.com/elastic/kibana/pull/164320?

Reply:
ah thanks for the extra eyes - you're right, the extra parameter isn't needed 👍🏻